CHANGELOG-BETA.md

```diff
@@ -4,6 +4,26 @@ _Beta versions are released on the `beta` stream. The latest beta can be install
 
 For the latest stable changes, see [CHANGELOG.md](./CHANGELOG.md).
 
+## [4.0.27-beta]
+
+### Added
+
+- Throws error when passing both dev and alpha options for MySky (#257). by @amberm31 in https://github.com/SkynetLabs/skynet-js/pull/352
+- Add Dependabot Auto-Merge action by @m-cat in https://github.com/SkynetLabs/skynet-js/pull/413
+
+### Changed
+
+- Mysky login flow fixes by @m-cat in https://github.com/SkynetLabs/skynet-js/pull/403
+- Change integration test to accommodate caching changes by @m-cat in https://github.com/SkynetLabs/skynet-js/pull/410
+
+### Dependencies updates
```

Comment:
We don't usually call out dependency updates, and most of these are minor updates. Can we remove this section so it doesn't clutter up the changelog?